### PR TITLE
#180 added property blacklist to Input.Model

### PR DESCRIPTION
**Pull request by mattcowen**
You can now ignore properties on a viewmodel using lambda expressions via a params on the Input.Model method of PageWriter.

### CODE REVIEW

**Comment by mattcowen** [comment]
Please review my PR. HTH

File: src/TestStack.Seleno.jmconfig [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿<?xml version="1.0" encoding="utf-8"?><Configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><DontShowAgainInSolution>false</DontShowAgainInSolution></Configuration>
```
Comment (robdmoore) [inline src/TestStack.Seleno.jmconfig]: What is this file?

**Comment by robdmoore** [comment]
I'm happy to merge this in; I like the change. We will need to document the breaking change in BREAKING_CHANGES.md (talking to fields by name rather than id) though. Are you able to have a go at that as part of this PR - keen not to delay the PR any further though?

FYI - for the future, it would be good if we can have separate PRs for separate changes - in this case there are two changes - property blacklist and changing to name. Just makes it easier to review and merge smaller changesets with a single logical change.

**Comment by mattcowen** [comment]
No problem, I will update the breaking changes shortly.

Ok will do in future. Makes sense.

Thanks
Matt

**Comment by robdmoore** [comment]
Nice. Ping back when you've updated breaking changes doc. Just mention the version number in the doc as 0.9.0 - I'll bump it to that

--
Rob Moore | Readify Principal Consultant, Technical Specialist (Microsoft Azure) | m +61 400 777 763 | e rob.moore@readify.net | w readify.net

**Comment by mattcowen** [comment]
This has been done

**Comment by robdmoore** [comment]
Thanks for the contribution mate!

**Pull request merged**